model-server/src/test/kotlin/org/modelix/model/server/handlers/ModelReplicationServerTest.kt

```diff
@@ -113,6 +117,81 @@ class ModelReplicationServerTest {
         }
     }
 
+    @Test
+    fun `responds with 404 when deleting a branch from a non-existent repository`() {
+        val url = "http://localhost/v2"
+        val storeClient = InMemoryStoreClient()
+        val modelClient = LocalModelClient(storeClient)
+        val repositoriesManager = RepositoriesManager(modelClient)
+        val modelReplicationServer = ModelReplicationServer(repositoriesManager, modelClient, InMemoryModels())
+
+        runWithTestModelServer(modelReplicationServer) {
```

Comment:
In this case you do not need to cunstruct a `modelReplicationServer`.
`runWithTestModelServer` uses `getDefaultModelReplicationServer()` to construct it.

Reply:
I had to do that to get access to the repositories manager, but I refactor the tests to provide that object to the block passed to `runWithTestModelServer`.

Reply:
better this way?

Reply:
Definitely. This is also a nice pattern for other tests.